src/Domain/ProductAttribute/Repositories/AttributeOptionRepository.php

```diff
@@ -49,4 +82,18 @@ public function update(string $id, AttributeOptionDto $dto): AttributeOption
 
         return $attributeOption;
     }
+
+    /**
+     * @param array<string, string> $translation
+     *
+     * @return array<string, string>
+     */
+    private function trimSpaces(array $translation): array
```

Comment:
In Laravel 11 Str helper class there is a nice implementation of trim that we could probably add to our own helpers instead of implementing it in single repository class (or we should upgrade to laravel 11...)

https://github.com/laravel/framework/blob/4b5ce19b3bbe1d317585d4b98da66fbb6e7357af/src/Illuminate/Support/Str.php#L1515

Reply:
I added own helper based on the one from laravel. The one from Laravel only removes spaces from the beginning and end of the string.